saleor/core/management/commands/populatedb.py

```diff
@@ -85,6 +86,13 @@ def sequence_reset(self):
             cursor.execute(commands.getvalue())
 
     def handle(self, *args, **options):
+        # set only our custom plugin to not call external API when preparing
+        # example database
+        settings.PLUGINS = [
+            "saleor.payment.gateways.dummy.plugin.DummyGatewayPlugin",
+            "saleor.payment.gateways.dummy_credit_card.plugin."
+            "DummyCreditCardGatewayPlugin",
```

Comment:
@mirumee/saleor Please change this to be on one line, these two look like separate entries and it will lead to a bug when someone edits the list and either adds a comma or inserts a line between those two.

Reply:
But then the line will be too long.